astroplan/tests/test_constraints.py

```diff
@@ -16,7 +16,7 @@
                            time_grid_from_range, SunSeparationConstraint,
                            MoonSeparationConstraint, MoonIlluminationConstraint,
                            TimeConstraint, LocalTimeConstraint, months_observable,
-                           max_best_rescale, min_best_rescale)
+                           max_best_rescale, min_best_rescale, AzimuthConstraint)
```

Comment:
bmorris3, Sorry that I had missed your reply from Oct.  I'll try to respond soon.  We are in a two-week run using astroplan for queue scheduling.  I've added several new constraints for our needs.  It's working well.